index.html

```diff
@@ -11,7 +11,7 @@
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
   <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet" />
   <link rel="icon" href="./images/profile-bg.png" type="image/png">
-  <title>Rwubakwanayo</title>
+  <title>Rwubakwanayo's Portfolio</title>
 </head>
 
 <body>
```

Comment:
Kindly change the contrast to be at least `4.5` for these failed elements because they failed in the contract check
![screencapture-aditus-io-button-contrast-checker-raw-githack-com-2022-03-29-at-15-37-29-060-2022-03-29-17_38_07](https://user-images.githubusercontent.com/24703171/160650683-f6fd1542-c90b-468f-bac9-159209212147.png)
You can check the contrast for your website using this [link](https://www.aditus.io/button-contrast-checker/)

Reply:
Hi @gemmen29 
I would like to ask if I have to change the Figma design color so that I can get at least 4.5 of color contrast

Reply:
Yes, you can but try the new color to be similar as much as possible to the Figma design.